### DISCUSSION TITLE
Are subject tokens available in NSCONFIG?

### DISCUSSION THREAD

**Discussion by bmpreston** [NightscoutFoundation/xDrip#2650] [General]
As I wrote, I would like to know if the subjects are in NSCONFIG, so I can control the tokens.

Thanks

**Comment by Navid200** [comment]
The Nightscout variables are in the nsconfig file.

Subject tokens?
What do you mean by that?

**Comment by bmpreston** [comment]
Unless I'm mistaken, when you add a subject, and give them a role under admin tools, this allows you to issue 'readable' or 'admin' level logins which are tokens, can you access and edit those tokens from the nsconfig?

**Reply by Navid200** [reply]
Those tokens are not in the nsconfig file.
The only way you can see them again is by going to the web site and going to the admin page.

**Reply by bmpreston** [reply]
Thank you

**Comment by bmpreston** [comment]
<img width="1659" alt="Screenshot 2023-01-26 at 08 23 02" src="https://user-images.githubusercontent.com/29456939/214846213-68793f44-3b4a-4f45-973c-1fbafaf3dfe3.png">
